.travis.yml

```diff
@@ -1,12 +1,15 @@
 sudo: false
 language: node_js
-node_js: stable
+node_js:
+    - '6'
 
 env:
   global:
     - CXX=g++-4.9
     - secure: "VDsxy30sE9ivdqoXkaKXo0czbS4brNpwKEIblu7f1gVLx7OD9pjTc78cdwrVbZDBYroSiYVYuUrLDjpVjH88lL/LxRrru3V0CIlAqqa+ssXcqycCaT/6ds+ZymuTTGRh+Mf12pIKO+yc8jTov2M7AzPJdpS+ORP5dImYyE3ex9s="
 
+script: ./test_type_defs.sh
```

Comment:
this would over-write the existing default script `npm test` - can we instead add the flow cli run to package.json scripts?